posts/2017-09-07-big-changes-for-a-small-org.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Preparing for Big Scale in a Small Shop Using Serverless"
+description: When a group of scrappy activists raise millions of dollars, they need an architecture that never turns a donor away
+date: 2017-09-07
+layout: Post
+thumbnail: https://movementvote.org/images/Movement-2018-logo-blue.png
```

Comment:
@victor-stone If you have a square image to go here, it'll make sure the thumbnail doesn't show up squished